tests/browser/heda_settings_accounts_and_contacts_edit_mode.robot

```diff
@@ -0,0 +1,64 @@
+*** Settings ***
+
+Resource        tests/HEDA.robot
+Suite Setup     Open Test Browser
+Suite Teardown  Delete Records and Close Browser
+
+*** Test Cases ***
+
+Go To Heda Home
+    Go To Heda Home
+    Select Frame			//iframe[contains(@id, "vfFrameId")]
+    Click Link				//a[contains(text(),'Accounts and Contacts')]
+    Wait Until Element Is visible	//a[contains(text(),'Accounts and Contacts')]
+
+Validate Edit Mode For Accounts and Contacts, Settings
+    
+    
+
+    #Go into Edit mode 
+    Click Element                   //div[@class='slds-button-group']//span[contains(text(), 'Edit')]
```

Comment:
Line 20 and 21 could be a python defined keyword Click HEDA Settings Button and the locator could accept an argument for the button label which would make this more generic and reusable throughout the suite